dev/reference/tidb-binlog-overview.md

```diff
@@ -1,26 +1,30 @@
 ---
-title: TiDB-Binlog Overview
-summary: Learn overview of the cluster version of TiDB-Binlog.
+title: TiDB Binlog Overview
+summary: Learn overview of the cluster version of TiDB Binlog.
 category: reference 
+<<<<<<< HEAD:dev/reference/tidb-binlog-overview.md
+aliases: ['/docs/tools/binlog/overview/','docs/tools/tidb-binlog-cluster/','/docs-cn/dev/reference/tools/tidb-binlog/overview/']
+=======
+>>>>>>> docs/master:dev/reference/tools/tidb-binlog/overview.md
```

Comment:
This looks like a merge is accidentally included in the commit.

Reply:
Fixed in https://github.com/pingcap/docs/pull/1251/commits/fce70f44743e56daebe705457b569c2fcedbb6a1